plugins/unreal/XJGame/Plugins/XJMusicPlugin/Source/XJMusicPlugin/Private/Test/XjTestActor.cpp

```diff
@@ -0,0 +1,32 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "Test/XjTestActor.h"
+#include "XjMusicInstanceSubsystem.h"
+#include <Kismet/GameplayStatics.h>
+#include <Sound/SoundBase.h>
+#include <Sound/SoundWave.h>
+#include <Components/AudioComponent.h>
+
+AXjTestActor::AXjTestActor()
+{
+	PrimaryActorTick.bCanEverTick = true;
+}
+
+void AXjTestActor::BeginPlay()
+{
+	Super::BeginPlay();
+	
+	UXjMusicInstanceSubsystem* XjMusicInstanceSubsystem = GetWorld()->GetGameInstance()->GetSubsystem<UXjMusicInstanceSubsystem>();
+	if (XjMusicInstanceSubsystem)
+	{
+		XjMusicInstanceSubsystem->RetrieveProjectsContent("D:/Dev/vgm/build");
```

Comment:
I'm curious about the function of this XjTestActor-- I have been using the PrototypeActor to test

Reply:
Also noting it is hard-coded here to the path on your disk, as opposed to how we now have the PrototypeActor using a UPROPERTY

Reply:
It's not that important actor. It's only to test the audio playing system, only used for this. That's why I didn't done there anything similiar